github-user-main/set-env/setEnvHandler.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable quote-props */
+import fs from 'fs'
+import os from 'os'
+import { IArgumentsSetEnv } from '../../@types/types'
+
+const tempPath = os.tmpdir()
+
+export const setEnvHandler = async ({ token, databaseUrl }: IArgumentsSetEnv) => {
+	try {
+		fs.writeFileSync(`${tempPath}/envParameters.json`, JSON.stringify({ token, databaseUrl }))
+		console.log('Environment variables were set')
```

Comment:
How probable is it for us to have a axios error here?

Reply:
Not possible at all haha 
This function is used only to set environment variables so the user can call githubUsers in any path.